x/evm/keeper/address.go

```diff
@@ -0,0 +1,33 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/sei-protocol/sei-chain/x/evm/types"
+)
+
+func (k *Keeper) SetAddressMapping(ctx sdk.Context, seiAddress sdk.AccAddress, evmAddress common.Address) {
```

Comment:
I assume this is for faster lookup. It seems for each account we would store a bidirectional mapping between seiAddress and evmAddress, that's a bit expensive. Is there no other way we can infer/convert/generate evmAddress from sei or vice versa?

Reply:
we can technically only store a EVM->Sei mapping, and use the Pubkey that we are already storing to derive the EVM address for query from Sei to EVM addr. Let me change it to that

Reply:
well actually we can't do what I described above, since we only store the 33-byte compressed version of the pubkey, whereas to restore the evm address we need the full 65-byte pubkey. To change it to store full pubkey would be a major migration, and also it doesn't save that much space compared to storing a additional direction of mapping